### PR TITLE
fix(FEC-10692): Youbora is missing some ad properties of post bumper

### PR DESCRIPTION
**Pull request by JonathanTGold**
### Description of the Changes

**issue:** Youbora is missing some ad properties of post bumper (in adObj)

**fix:** the trigger for youbora request is the adstarted event, 
          and initialisation of the adObj is just on adloaded event, 
(and the reset of the adObj is on adcompleted event),   so the fix is to dispatch adloaded each time 
of dispatching adstartd **in case the current starting bumper is post-bumper**

solves: (FEC-10692)

### CheckLists

- [ ] changes have been done against master branch, and PR does not conflict
- [ ] new unit / functional tests have been added (whenever applicable)
- [ ] test are passing in local environment
- [ ] Travis tests are passing (or test results are not worse than on master branch :))
- [ ] Docs have been updated

### CODE REVIEW

File: src/bumper.js [hunk 1]
```diff
@@ -221,7 +221,7 @@ class Bumper extends BasePlugin implements IMiddlewareProvider, IAdsControllerPr
 
   onEnded(): void {
     if (this._adBreak) {
-      this._state = BumperState.LOADED;
+      this._state = BumperState.IDLE;
```
Comment (yairans) [inline src/bumper.js:224]: this means you will always reload the post bumper although it already loaded

**Review by yairans — CHANGES_REQUESTED** [review]

File: src/bumper.js [hunk 2]
```diff
@@ -515,6 +517,8 @@ class Bumper extends BasePlugin implements IMiddlewareProvider, IAdsControllerPr
         this._bumperVideoElement.src = this.config.url;
         this._bumperVideoElement.setAttribute('playsinline', '');
       }
+    } else if (this._bumperState === BumperState.LOADED) {
+      this.dispatchEvent(EventType.AD_LOADED, {ad: this._getAd()});
```
Comment (yairans) [inline src/bumper.js:521]: so you will get this line multiple times when only postroll exists no?

Comment (JonathanTGold) [reply src/bumper.js:521]: מot even once, according to what I checked

Comment (JonathanTGold) [reply src/bumper.js:521]: not even once, according to what I checked

File: src/bumper.js [hunk 3]
```diff
@@ -239,6 +239,9 @@ class Bumper extends BasePlugin implements IMiddlewareProvider, IAdsControllerPr
       this.playOnMainVideoTag() && (this._state = BumperState.IDLE);
       this.initBumperCompletedPromise();
       this.play();
+      if (this._bumperState === BumperState.LOADED && this.config.position.length === 2) {
```
Comment (yairans) [inline src/bumper.js:242]: why `this._bumperState === BumperState.LOADED` is not enough?

**Review by yairans — APPROVED** [review]

**Review by yairans — APPROVED** [review]

**Pull request merged**